src/test_workflow/integ_test/integ_test_suite_opensearch_dashboards.py

```diff
@@ -81,6 +94,38 @@ def __setup_cluster_and_execute_test_config(self, config: str) -> int:
             os.chdir(self.work_dir)
             return self.execute_integtest_sh(endpoint, port, security, config)
 
+    def multi_execute_integtest_sh(self, cluster_endpoints: list, security: bool, test_config: str) -> int:
+        script = ScriptFinder.find_integ_test_script(self.component.name, self.repo.working_directory)
```

Comment:
Add tests for this asserting that `self.repo` is always `functionalTestDashboards` for opensearch dashboards